app/components/create_update_form_component.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class CreateUpdateFormComponent < ApplicationComponent
```

Comment:
We should probably either make a generic form component then wrap it with a create form and update form or just have two different ones so we can better match the component mindset.